devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy

```diff
@@ -29,12 +29,13 @@ metadata {
         capability "Sensor"
         capability "Contact Sensor"
         capability "afterguide46998.peopleCounter"
+        capability "afterguide46998.inOutDirection"
```

Comment:
Do other devices from this DTH (fingerprints that were added previously) support this new custom capability ? `afterguide46998.inOutDirection`

Reply:
There is no problem simply adding a status.